README.md

```diff
@@ -46,19 +46,19 @@ Click the below links to load a specific branch from this repository into codesa
 
 - [Input 01: Display a basic input text](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/cp/kodiri/beginner/input/01)
 - [Input 02: Display an input text with dynamic data](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/cp/kodiri/beginner/input/02)
-- [Input 03: Display a textarea with dynamic data](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/cp/kodiri/beginner/input/03)
+- [Input 03: Display a textarea wi<!-- t -->h dynamic data](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/cp/kodiri/beginner/input/03)
```

Comment:
what is this achieving?

Reply:
Just some clumsy typing in vscode. Fixed.